src/gba/supervisor/config.c

```diff
@@ -166,7 +166,7 @@ void GBAConfigDirectory(char* out, size_t outLength) {
 	char* home = getenv("HOME");
 	snprintf(out, outLength, "%s/.config", home);
 	mkdir(out, 0755);
-	snprintf(out, outLength, "%s/.config/%s", home, binaryName);
+        snprintf(out, outLength, "%s/.config/%s", home, binaryName);
```

Comment:
Might want to fix this so the indentation matches. That would remove this from the diff entirely.